apps/reaper/lib/reaper/decoder/json.ex

```diff
@@ -47,10 +47,18 @@ defmodule Reaper.Decoder.Json do
 
     {:ok, data}
   rescue
-    error -> {:error, error}
+    error ->
+      parse_json_file_query_errors(error)
   end
 
   def truncate_file_for_logging(filename) do
     File.stream!(filename, [], 1000) |> Enum.at(0)
   end
+
```

Comment:
So basically you're just matching on Jaxon parse errors on unexpected end_array and returning an ok tuple with an empty array.